packages/server/src/cli.ts

```diff
@@ -166,6 +166,24 @@ if (isCli) {
 		password = buffer.toString("hex");
 	}
 
+	// If CLI password was provided, obsfucate password from process title
+	if (options.password) {
+		const parts = [process.title];
+		for (let i = 2; i < process.argv.length; i++) {
+			if (process.argv[i].startsWith("--password=")) {
+				parts.push(process.argv[i].replace(/=.*/, "=****"));
+			} else if (process.argv[i] === "--password") {
+				parts.push(process.argv[i++], "****")
+			} else if (process.argv[i] === "--") {
```

Comment:
Another unfortunate edge case comes to mind:

```
code-server --data-dir -- password pass
```

That is, `--` is only interpreted as the end of options if it isn't the argument for another option.

Reply:
It's possible we should use an environment variable and/or configuration file instead of a command line argument for the password. Or possibly a prompt, although I'm not really keen on that solution.